editor/editor_zoom_widget.cpp

```diff
@@ -201,5 +205,5 @@ EditorZoomWidget::EditorZoomWidget() {
 
 	_update_zoom_label();
 
-	add_theme_constant_override("separation", Math::round(-8 * EDSCALE));
+	add_theme_constant_override("separation", -1);
```

Comment:
Shouldn't this be scaled with `EDSCALE` too?

Reply:
Nah. It actually should be 0, but that didn't work for some reason when I tried it at first. I'll test it again.

Reply:
Changed to 0.